.eslintignore

```diff
@@ -7,3 +7,4 @@ coverage/*
 dist/*
 node_modules/*
 script/utils/dist/*
+test/e2e/performance
```

Comment:
:question: shouldn't we restrict this to the data subfolder? otherwise we wont't run on the performance test code

Reply:
sure
I forgot to change it after moving tests under this directory
done

Reply:
cool